apps/explorer/src/app/routes/home/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { StatsManager } from '@vegaprotocol/mainnet-stats-manager';
+import { StatsManager } from '@vegaprotocol/network-stats';
```

Comment:
question (blocking): why is the package called network-stats when directory is called stats?

Reply:
urgh - naming is hard! I considered calling the package the same thing as the app, because the app is essentially just a wrapper around the component. But that didn't feel like a great idea, so opted for something that was different but the same, if that makes sense :)